src/video_core/renderer_opengl/renderer_opengl.cpp

```diff
@@ -1090,11 +1104,22 @@ void RendererOpenGL::TryPresent(int timeout_ms) {
 void RendererOpenGL::UpdateFramerate() {}
 
 void RendererOpenGL::PrepareVideoDumping() {
+    auto* mailbox = static_cast<OGLVideoDumpingMailbox*>(frame_dumper.mailbox.get());
+    {
+        std::unique_lock(mailbox->swap_chain_lock);
```

Comment:
```suggestion
        std::unique_lock lock(mailbox->swap_chain_lock);
```
Otherwise the lock would be immediately destructed